packages/frontend/src/api/useDialogs.tsx

```diff
@@ -98,6 +106,13 @@ export function mapAutocompleteDialogsDtoToInboxItem(
   });
 }
 
+export const getSeenByLabel = (seenBy): { isSeenByYou: string; seenByOthersCount: number } => {
```

Comment:
input param `seenBy` misses a type and a bit confused about the return type for this function. Isn't better that isSeenByYou is a `boolean`, and not a string?


```
type SeenByItem = { isCurrentEndUser: boolean };

export const getSeenByLabel = (
  seenBy: DialogByIdFieldsFragment['seenBy'] ): { isSeenByYou: boolean; seenByOthersCount: number } => {
  const isSeenByYou = seenBy.some((item) => item.isCurrentEndUser);
  const seenByOthersCount = seenBy.filter((item) => !item.isCurrentEndUser).length;

  return { isSeenByYou, seenByOthersCount };
};
```